scifight/models.py

```diff
@@ -58,6 +58,14 @@ class Meta:
     def __str__(self):
         return self.name
 
+    def clean_fields(self, exclude=None):
+        # Without this option does not work unique_togetner.
+        super().clean_fields(exclude)
+
+        if self.slug not in exclude:
+            if self.slug == "":
+                self.slug = None
+
```

Comment:
Ein moment, but:
```python
# Force Django to always store SQL NULL for slug instead of an empty string,
# thus making 'unique_together' work again. (Note: in SQL NULLs are treated
# as unique values.)
if "slug" not in exclude:
    if self.slug == "":
        self.slug = None
```